README.md

```diff
@@ -8,7 +8,7 @@ Inspired by [fennifith/AndroidUtils](https://github.com/fennifith/AndroidUtils)
 ## Feature
 This library provides the collection of utils classes.
 
-<img src="https://github.com/applibgroup/Instalike/blob/master/screenshots/instalikeview.gif" width="256">
+<img src="https://github.com/vidyaakbar/OHOSUtils/blob/main/screenshots/hmosutilsgif.gif" width="256">
```

Comment:
The gif link must point to the one that is present in the project's screenshots folder. Currently this is pointing to a different github account's screenshot folder.. Please check

Reply:
The content in the current gif present in the ReadMe is not visible properly.. Please replace it with a  clearer gif.

Reply:
Done